### DISCUSSION TITLE
UI: Menus

### DISCUSSION THREAD

**Discussion by tmedwards** [tmedwards/sugarcube-3#38] [Ideas]
The core in-play menu is being removed from the UI bar, which is becoming a blank canvas for user content.  The new in-play menu will probably open into a dialog or something to that effect.

I also plan to create a startup menu that'll have a few more items, like a continue button to load the most recent auto save (maybe save in general).

Thoughts?

**Comment by ChapelR** [comment]
As long as the start-up menu is optional and configurable it sounds like a great idea. I think optional timed/press any key splash screens will also be a good addition. Forgive me if that's in the issues, I feel like it's been discussed before.

**Comment by greyelf** [comment]
While I agree a [optional] start-up menu could be a good thing, some end-users may assume they can play audio on it, which will possible be an issue if the end-user hasn't interacted with something before its shown.
